app/views/directives/notifications/notification-counter.html

```diff
@@ -0,0 +1,13 @@
+<a
+  ng-if="!$ctrl.hide"
+  href=""
+  ng-click="$ctrl.onClick()"
+  class="nav-item-iconic relative">
+  <div
+    ng-if="$ctrl.showNewNotificationIndicator"
+    class="notification-counter notification-counter-offset"
+    row main-axis="center" cross-axis="center">
+    <span><!--{{countDisplay}}--></span>
+  </div>
```

Comment:
OK, mind removing out the commented out expression (and any unnecessary markup)?

We should add some screenreader text as well per my comment below.